api/src/services/agencies/agencies.ts

```diff
@@ -35,6 +36,40 @@ export const deleteAgency: MutationResolvers['deleteAgency'] = ({ id }) => {
   })
 }
 
+export const agenciesAvailableForUpload = async () => {
+  try {
+    const agencies = []
+    const user = await db.user.findFirst({
+      where: { id: context.currentUser.id },
+    })
+
+    if (!user) {
+      logger.error('User not found')
+      return
+    }
+
+    const agency = await db.agency.findFirst({
```

Comment:
I think we can simplify this a bit further since `currentUser` includes `agency` and we're already looking up the user from the DB when we retrieve `currentUser` - https://github.com/usdigitalresponse/cpf-reporter/blob/main/api/src/lib/auth.ts#L66

```js
async function agenciesAvailableForUpload() {
  if (currentUser.role === ..STAFF) {
      return [currentUser.agency]
  } else {
      return await agenciesByOrganization({ organizationId: currentUser.agency.organizationId })
  }
}
```

Reply:
Reopened the PR with this change here: https://github.com/usdigitalresponse/cpf-reporter/pull/337